core/src/main/java/apoc/util/FileUtils.java

```diff
@@ -42,6 +42,7 @@
     public enum SupportedProtocols {
         http(true, null),
         https(true, null),
+        ftp(true, null),
```

Comment:
This change is for consistency with the database. At the moment we don't let import a csv from an ftp server, we consider the url a file instead and that's blocked in Aura for example.

Any concerns doing this?

Reply:
Unfortunately `ftps` is not recognised by java URL, so we need to remove it from here